woocommerce/wc-admin
# Fix OBW when new home screen is enabled.

This PR seeks to restore the OBW when the new home screen is enabled.

It appears this was broken in https://github.com/woocommerce/woocommerce-admin/pull/4169.

### Detailed test instructions:

- On `master`, start dev build (`npm start`)
- Enable the profile wizard (WooCommerce > Orders, Help > Setup wizard)
- Go to WooCommerce > Home
- Verify placeholder "hello world" is shown, no OBW
- Go to Analytics > Dashboard
- Verify OBW is shown
- On this branch, start dev build (`npm start`)
- Go to WooCommerce > Home
- Verify OBW is shown
- Go to Analytics > Dashboard
- Verify dashboard is shown, no OBW
